icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -4883,6 +4897,104 @@ void RBBITest::TestDebugRules() {
 #endif
 }
 
+void RBBITest::testTrieStateTable(int32_t numChar, UCPTrieValueWidth expectedTrieWidth,
+                                  int32_t expectedStateRowBits) {
+    // Text are duplicate characters from U+4E00 to U+4FFFF
+    UnicodeString text("");
+    for (UChar c = 0x4e00; c < 0x5000; c++) {
+        text.append(c).append(c);
+    }
+    // Generate rule which will caused length+4 character classes and
+    // length+3 states
+    UnicodeString rules("!!quoted_literals_only;");
```

Comment:
Use u"!!quoted_literals_only;"
Without the u it will go through code page conversion on some platforms.

Reply:
done